G/Graphviz/bundled/patches/gvpr-build-native-mkdefs.patch

```diff
@@ -0,0 +1,18 @@
+--- a/lib/gvpr/Makefile.in
++++ b/lib/gvpr/Makefile.in
+@@ -702,9 +702,14 @@
+ 	echo " rm -f" $$list; \
+ 	rm -f $$list
+ 
++$(mkdefs_OBJECTS): $(mkdefs_SOURCES)
++	$(CC_FOR_BUILD) $(CFLAGS) $(LDFLAGS) -c $<
++
+ mkdefs$(EXEEXT): $(mkdefs_OBJECTS) $(mkdefs_DEPENDENCIES) $(EXTRA_mkdefs_DEPENDENCIES) 
+ 	@rm -f mkdefs$(EXEEXT)
+-	$(AM_V_CCLD)$(LINK) $(mkdefs_OBJECTS) $(mkdefs_LDADD) $(LIBS)
++	$(AM_V_CCLD)$(LIBTOOL) $(AM_V_lt) --tag=CC $(AM_LIBTOOLFLAGS) \
```

Comment:
For the record, what I did here was to replace `$(LINK)` with its value in this Makefile and then replace `$(CCLD)` with `$(CC_FOR_BUILD)`, which is the native C compiler.  In addition I added a rule above to build `mkdefs.o` with the native compiler as well.